src/demo/components/TextboxLayout.tsx

```diff
@@ -19,7 +19,7 @@ const TextboxLayout = (props: TextboxLayoutProps) => {
         }}
       />
       <label htmlFor="inputField" className="textbox-label">
-        {props.label}
+        tes
```

Comment:
keep the {props.label} here

Reply:
I removed this component. I no longer need this.

Reply:
Done